src/Ng/Jobs/EndpointMonitoringJob.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using NuGet.Services.Configuration;
+using NuGet.Services.Metadata.Catalog;
+using NuGet.Services.Metadata.Catalog.Monitoring;
+using NuGet.Protocol.Core.Types;
+using NuGet.Protocol;
+using NuGet.Services.Metadata.Catalog.Monitoring.Validation.Test;
+using System.Linq;
```

Comment:
If they are already out of order, I don't think VS does the full fix-up. There is a `Remove and Sort Usings` button you can add to your UI to do it easily. There is also an option in VS options to put System.* first.